Misc/NEWS.d/next/Library/2021-05-18-00-17-21.bpo-27334.32EJZi.rst

```diff
@@ -0,0 +1,2 @@
+The :mod:`sqlite3` context manager now releases the database lock if the
```

Comment:
This description sounds too low-level to me. Maybe just that a rollback is run if the commit failed?

Reply:
You can keep the sentence about the "database lock" in a parenthesis ;-)

Reply:
Yes, this is too low level. I'll rewrite asap :)